src/blk/zoned/HMSMRDevice.cc

```diff
@@ -412,6 +412,14 @@ void HMSMRDevice::_detect_vdo()
   return;
 }
 
+// Will reset the write pointer of all zones from start to end(both inclusive)
+bool HMSMRDevice::reset_zones(uint64_t zone_num_range_start, uint64_t zone_num_range_end) {
```

Comment:
Return int to be consistent with the remaining functions.

Nit: Omit range from parameter names.

Reply:
Updated.
Thank you.